go/coordinator/internal/memberlist_manager/memberlist_manager.go

```diff
@@ -0,0 +1,96 @@
+package memberlist_manager
```

Comment:
So for now we just watch the pods but we don't create any of them correct?

Reply:
yes